packages/babel-plugin-transform-flow-comments/package.json

```diff
@@ -16,10 +16,12 @@
     "@babel/plugin-syntax-flow": "^7.2.0"
   },
   "peerDependencies": {
-    "@babel/core": "^7.0.0-0"
+    "@babel/core": "^7.0.0-0",
+    "@babel/generator": "^7.0.0-0"
```

Comment:
Requiring users to install a new package is a breaking change.
I believe you can import it from `@babel/core`.

Reply:
interestingly `@babel/core` does not export `generate` function from `@babel/generator`, should I export it out from `@babel/core`?

Reply:
Oh I didn't know that.
If we added a `generate` export, we couldn't use it in this plugin anyway because it needs to be compatible with `@babel/core@^7.0.0`. We can add `@babel/generator` to the dependencies of this plugin.

Reply:
erm. then is it okay to use `@babel/generator` as `peerDependency`? since it's `@babel/core`'s dependency

Reply:
No, because peer dependencies must be explicitly installed by the user, otherwise npm will log a warning